common/djangoapps/third_party_auth/provider.py

```diff
@@ -39,7 +39,7 @@ def enabled(cls):
     @classmethod
     def accepting_logins(cls):
         """Returns list of providers that can be used to initiate logins currently"""
-        return [provider for provider in cls.enabled() if provider.accepts_logins]
+        return [provider for provider in cls.enabled() if provider.display_for_login]
```

Comment:
If that's a lot of work/change then we can refrain from doing so -- I just would like it to be super clear what is going on.  Just because a provider accepts logins doesn't mean that it is being displayed anywhere.